src/backend/core/tests/teams/test_core_api_teams_update.py

```diff
@@ -188,3 +190,95 @@ def test_api_teams_update_administrator_or_owner_of_another():
     team.refresh_from_db()
     team_values = serializers.TeamSerializer(instance=team).data
     assert team_values == old_team_values
+
+
+@responses.activate
+def test_api_teams_update_authenticated_owner_resource_server(
```

Comment:
I was expecting tests about adding/removing service providers from a team for authenticated admin/owners of the team?